tests/unit/amo/components/TestAddonMeta.js

```diff
@@ -27,6 +31,23 @@ describe(__filename, () => {
     );
   }
 
+  function mountRender({
```

Comment:
I don't think you should need `mount` for this test; obviously you'll want to change it to be in `AddonMoreInfo`, but this should work with `shallowUntilRender`.